python/ray/train/torch/torch_trainer.py

```diff
@@ -103,65 +106,106 @@ def train_loop_per_worker():
             from ray.air import session, Checkpoint
             from ray.train.torch import TorchTrainer
             from ray.air.config import ScalingConfig
+            from ray.air.config import RunConfig
+            from ray.air.config import CheckpointConfig
 
+            # Define NN layers archicture, epochs, and number of workers
             input_size = 1
-            layer_size = 15
+            layer_size = 32
             output_size = 1
-            num_epochs = 3
+            num_epochs = 180
+            num_workers = 3
 
+            # Define your network structure
             class NeuralNetwork(nn.Module):
                 def __init__(self):
                     super(NeuralNetwork, self).__init__()
                     self.layer1 = nn.Linear(input_size, layer_size)
-                    self.relu = nn.ReLU()
                     self.layer2 = nn.Linear(layer_size, output_size)
 
                 def forward(self, input):
-                    return self.layer2(self.relu(self.layer1(input)))
```

Comment:
I would either keep the ReLU layer here or have only one linear layer -- composing two linear layers doesn't do anything and it would likely be confusing to users :)